Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -486,7 +486,7 @@ BindablePropertyContext GetContext(BindableProperty property)
 			for (var i = 0; i < properties.Count; i++)
 			{
 				BindablePropertyContext context = properties[i];
-				if (ReferenceEquals(context.Property, property))
+				if (context != null && ReferenceEquals(context.Property, property))
```

Comment:
@StephaneDelcroix I can't reproduce this problem because it is floating and it be not always.

Can't I change bindable properties of view in second thread? If I can't that why do I changed bindable properties and I didn't get exceptions before? I got GetContext exception only when I navigate to page.